docs/filter-audit-log-filter-files.md

```diff
@@ -0,0 +1,49 @@
+# Filter the Audit Log Filter logs
+
+The audit filter log filtering is based on rules. The filter rule definition has the ability to include or exclude events based on the following attributes:
+
+* User account
+* Audit event class
+* Audit event subclass
+* Audit event fields (for example, `COMMAND_CLASS` or `STATUS`)
+
+You can define multiple filters and assign any filter to multiple accounts. You can also create a default filter for specific user accounts. The filters are defined using function calls. After the filter is defined, the filter is stored in `mysql` system tables. 
+
+## Audit Log Filter functions
+
+The Audit Log filter functions require `AUDIT_ADMIN` or `SUPER` privilege. 
+
+The following functions are used for rule-based filtering:
+
+| Function | Description | Example |
+|---|---|---|
+| audit_log_filter_flush() | Manually flush the filter tables | `SELECT audit_log_filter_flush()`
+| audit_log_filter_set_filter() | Defines a filter | `SELECT audit_log_filter_set_filter('log_connections','{ "filter":{}}'`')
+| audit_log_filter_remove_filter() | Removes a filter |
+| audit_log_filter_set_user() | Assigns a filter to a specific user account |
+| audit_log_filter_remove_user() | Removes the filters from a specific user account |
+
+Using a SQL interface, you can define, display, or modify audit log filters. The filters are stored in the `mysql` system database. 
+
+A read-only variable, `audit_log_session_filter_id()`, signals if a filter is assigned to a specific session.
```

Comment:
"A read-only variable" - Return value of audit_log_session_filter_id() function )

Reply:
updated